config.json

```
@@ -867,6 +867,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 4
+      },
+      {
+        "slug": "variable-length-quantity",
+        "name": "Variable Length Quantity",
+        "uuid": "66e4bb61-076a-4adb-b7a0-b216196e699e",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

```suggestion
        "difficulty": 5
```
This is a medium difficulty exercise